playbooks/files/run-build-hook.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+BUILD=$1
+
+if [[ -f $BUILD/.nexcess-app.yml ]]; then
+    cd $BUILD
+    cat $BUILD/.nexcess-app.yml | shyaml get-value hooks.build | bash -x
```

Comment:
Yes, although we don't have a "nice" way to communicate hook error messages back to the user yet, which will be important, probably pretty soon.  Getting friendly output from the result of an ansible run (task, or whole playbook) is a thing I haven't figured out a good way to do yet.